pnpm-lock.yaml

```diff
@@ -1,91 +1,100 @@
-    specifier: ^3.5.1
-    version: 3.5.1(vite@5.4.0)
+    specifier: ^3.5.2
+    version: 3.5.2(vite@5.4.8)
 
 devDependencies:
   '@assistant-ui/react':
-    specifier: ^0.5.27
-    version: 0.5.27(@types/react-dom@18.3.0)(@types/react@18.3.3)(react-dom@18.3.1)(react@18.3.1)(tailwindcss@3.4.9)
+    specifier: ^0.5.70
+    version: 0.5.70(@types/react-dom@18.3.0)(@types/react@18.3.10)(react-dom@18.3.1)(react@18.3.1)(tailwindcss@3.4.13)
   '@assistant-ui/react-markdown':
-    specifier: ^0.2.8
-    version: 0.2.8(@assistant-ui/react@0.5.27)(@types/react-dom@18.3.0)(@types/react@18.3.3)(react-dom@18.3.1)(react@18.3.1)(tailwindcss@3.4.9)
+    specifier: ^0.2.18
+    version: 0.2.18(@assistant-ui/react@0.5.70)(@types/react-dom@18.3.0)(@types/react@18.3.10)(react-dom@18.3.1)(react@18.3.1)(tailwindcss@3.4.13)
   '@assistant-ui/react-syntax-highlighter':
     specifier: ^0.0.8
-    version: 0.0.8(@assistant-ui/react-markdown@0.2.8)(@assistant-ui/react@0.5.27)(@types/react-syntax-highlighter@15.5.13)(@types/react@18.3.3)(react-syntax-highlighter@15.5.0)(react@18.3.1)
+    version: 0.0.8(@assistant-ui/react-markdown@0.2.18)(@assistant-ui/react@0.5.70)(@types/react-syntax-highlighter@15.5.13)(@types/react@18.3.10)(react-syntax-highlighter@15.5.0)(react@18.3.1)
   '@eslint/js':
-    specifier: ^9.8.0
-    version: 9.8.0
+    specifier: ^9.11.1
+    version: 9.11.1
   '@types/node':
-    specifier: ^22.2.0
-    version: 22.2.0
+    specifier: ^22.7.4
+    version: 22.7.4
   '@types/react':
-    specifier: ^18.3.3
-    version: 18.3.3
+    specifier: ^18.3.10
+    version: 18.3.10
   '@types/react-dom':
     specifier: ^18.3.0
     version: 18.3.0
   '@types/react-syntax-highlighter':
     specifier: ^15.5.13
     version: 15.5.13
   '@vitejs/plugin-react':
-    specifier: ^4.3.1
-    version: 4.3.1(vite@5.4.0)
+    specifier: ^4.3.2
+    version: 4.3.2(vite@5.4.8)
   autoprefixer:
     specifier: ^10.4.20
-    version: 10.4.20(postcss@8.4.41)
+    version: 10.4.20(postcss@8.4.47)
+  axios:
+    specifier: ^1.7.7
+    version: 1.7.7
   eslint:
-    specifier: ^9.8.0
-    version: 9.8.0
+    specifier: ^9.11.1
+    version: 9.11.1
   eslint-plugin-react-hooks:
-    specifier: ^5.1.0-rc.0
-    version: 5.1.0-rc.0(eslint@9.8.0)
+    specifier: 5.1.0-rc-fb9a90fa48-20240614
+    version: 5.1.0-rc-fb9a90fa48-20240614(eslint@9.11.1)
   eslint-plugin-react-refresh:
-    specifier: ^0.4.9
-    version: 0.4.9(eslint@9.8.0)
+    specifier: ^0.4.12
+    version: 0.4.12(eslint@9.11.1)
   globals:
-    specifier: ^15.9.0
-    version: 15.9.0
+    specifier: ^15.10.0
+    version: 15.10.0
   postcss:
-    specifier: ^8.4.41
-    version: 8.4.41
+    specifier: ^8.4.47
+    version: 8.4.47
   posthog-js:
-    specifier: ^1.155.0
-    version: 1.155.0
+    specifier: ^1.166.1
+    version: 1.166.1
   react-syntax-highlighter:
     specifier: ^15.5.0
     version: 15.5.0(react@18.3.1)
   remark-gfm:
     specifier: ^4.0.0
     version: 4.0.0
```

Comment:
Using release candidate version `5.1.0-rc-fb9a90fa48-20240614` of `eslint-plugin-react-hooks` may introduce instability. Use stable version.